build.gradle.kts

```diff
@@ -44,6 +44,7 @@ dependencies {
   testImplementation("org.mockito:mockito-inline:4.4.0")
   testImplementation("io.swagger.parser.v3:swagger-parser-v3:2.0.31")
   testImplementation("org.testcontainers:localstack:1.16.3")
+  testImplementation("org.awaitility:awaitility:4.2.0")
```

Comment:
I dont know if we use a different library to test async behaviour, but I've used this in previous projects 👍 
There are tests later in this PR where we need to wait for something to have happened, hence needing this (or similar) library

Reply:
Good call, I really like awaitility.